org-brain.el

```diff
@@ -266,6 +274,42 @@ Ignores \"dotfiles\"."
     (directory-files-recursively
      org-brain-path (format "^[^.].*\\.%s$" org-brain-files-extension))))
 
+(defun org-brain-replace-links-with-visible-parts (raw-str)
+  "Return `str', with all bracketed links replaced with the link
+descriptions only (if applicable, otherwise shows the link destination)."
+  (let ((ret-str "")
+		(start 0)
```

Comment:
Please use spaces instead of tabs, since tabs makes the indentation look strange (at least on github and on my computer).

Reply:
Fixed. And, I've fixed my emacs config so the indentation doesn't get messed up again.